cms/pdo.class.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * PDO Class
+ *
+ * PHP version 5
+ * 
+ * @category CMS
+ * @package  Pragyan
+ * @class    pdo Class which uses PHP Data Objects to access database
+ * @author   Vignesh Manix <vigneshmanix@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl.html GNU GPL V3
+ * @link     https://github.com/delta/pragyan/
+ * For more details, see README
+ */
+
+if (!defined('__PRAGYAN_CMS')) {
+    header($_SERVER['SERVER_PROTOCOL'].' 403 Forbidden');
+    echo "<h1>403 Forbidden</h1><h4>You are not authorized to access the page.</h4>";
+    echo '<hr/>'.$_SERVER['SERVER_SIGNATURE'];
+    exit(1);
+}
+
+class pdodb
+{
+    private $_pdo;   // @object The PDO object
+    private $_stmt;  // @object PDO statement object
+    private $_isconnected; // @bool connected to database
+    private $_params; // @array Paramaters of query
+
+
+    public function __construct()  // Constructor sets $isconnected, connects to db and creates params array
+    {
+        $this->_isconnected = false;
+        $this->_connect();
+        $this->_params=array();
+    }
+
+
+    private function _connect()  // connect tries to connect 
+    {
+        if (MYSQL_SERVER=='localhost') {
```

Comment:
Do we need the (if, else_ condition ? localhost will work, why do we need to define it as 127.0.0.1 ?